doc/api/v0.yaml

```diff
@@ -315,35 +361,11 @@ components:
             proposer_url:
               type: string
               description: URL to a web resource with details about the author of the proposal.
-        chain_proposal_id:
-          type: string
-          description: Identifier of the proposal on the blockchain.
-        chain_proposal_index:
-          type: integer
-          format: int64
-          description: Index of the proposal in the vote plan.
-        chain_vote_options:
-          description: Map of named vote options to choice indices.
-          type: object
-        chain_voteplan_id:
-          type: string
-          description: Identifier of the vote plan this proposal belongs to.
-        chain_vote_start_time:
-          type: string
-          format: date-time
-          description: Date and time for the start of voting on this proposal's vote plan.
-        chain_vote_end_time:
-          type: string
-          format: date-time
-          description: Date and time for the start of voting on this proposal's vote plan.
-        chain_committee_end_time:
-          type: string
-          format: date-time
-          description: Date and time for the end of tallying on this proposal's vote plan.
-        chain_voteplan_payload:
-          type: string
-          description: |
-            Whether the voting is done using the public or the privacy-preserving protocol.
+        chain_info:
+          type: array
```

Comment:
So you would use voting groups as indexes? I think it can be quite inefficient if we deal with voting groups >> 0. A map would be better suited for this